frontend/src/component/common/DialogFormTemplate/ConfigButtons/ToggleConfigButton.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ConfigButtonProps } from './ConfigButton';
+import { ButtonLabel, StyledTooltipContent } from './ConfigButton.styles';
+import { TooltipResolver } from 'component/common/TooltipResolver/TooltipResolver';
+import { Button } from '@mui/material';
+
+type ToggleConfigButtonProps = Pick<
```

Comment:
I feel like you are overusing the ConfigButtonProps props. I feel that you are picking it from a familiar type, but there is actually no reason.

1. It is hard to track to instantly see what types these are.
2. All properties from `button` (for example **additionalTooltipContent**) are not used, so it might be confusing to know.  If a property is added to button, there is no motivation to implement it in all of the files that is using it.

Reply:
Ah, good point! I might've missed the additional tooltip content. I'll make sure to add that.

As far as the naming goes, sure. What would you call it? Would you wanna change the entire component name?

Reply:
Done in 6531c38038cfedaa1aa816ea2a2e1395061e080f